lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -26,7 +26,8 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
        secrets: secrets,
        show_key: show_key?,
        secret_name: secret_name,
-       secret_value: secret_value
+       secret_value: secret_value,
+       hub_metadata: Provider.to_metadata(assigns.hub)
```

Comment:
Should we subscribe to events, so that if the hub connects we update the status?

Reply:
Yes, good catch!